exporter/server.go

```diff
@@ -97,6 +104,68 @@ func multiTargetHandler(serverMap ServerMap) http.HandlerFunc {
 	}
 }
 
+// OverallTargetsHandler is a handler to scrape all the targets in one request.
+// Adds instance label to each metric.
+func OverallTargetsHandler(exporters []*Exporter, logger *logrus.Logger) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		seconds, err := strconv.Atoi(r.Header.Get("X-Prometheus-Scrape-Timeout-Seconds"))
+		// To support also older ones vmagents.
+		if err != nil {
+			seconds = 10
```

Comment:
Helpful to log this? So folks don't scratch their heads when their set timeout isn't working.

Reply:
logged that as a debug message